File: Modul 2/Modul 2 Bahasa C/PRAK202-2410817310007-MuhammmadNaufalAbdillah.c
#include <stdio.h>
int main(){
    float P; float K;
    printf("Masukan NIlai Pertama : ");
    scanf("%f", &P);
    printf("Masukan Nilai Kedua : ");
    scanf("%f", &K);

    float Total1 =P+K;
    if (P==(int)P){
        printf("Hasil dari penjumlahan nilai pertama \"%d\" dan nilai kedua \"%.1f\" adalah \"%.2f\"\n\n",(int)P,K,Total1);
    }
    else{
        printf("Hasil dari penjumlahan nilai pertaama \"%.2f\" dan nilai kedua \"%.1f\" adlah \"%.2f\"\n\n",P,K,Total1);
    }
}

File: Modul 2/Modul 2 Bahasa C/Modul 2 Bahasa Python/PRAK202-2410817310007-MuhammmadNaufalAbdillah.py
P=float(input("masukan NIlai Pertama :"))
K=float(input("Masukan Nilai kedua :"))
hasil=P+K
if P.is_integer():
    print(f"Hasil dari penjumlahan nilai pertama \"{int(P)}\" dan nilai kedua \"{K:.1f}\" adalah \"{hasil:.2f}\" ")
else:
    print(f"Hasil dari penjumlahan nilai pertama \"{P:.2f}\" dan nilai kedua \"{K:.1f}\" adalah \"{hasil:.2f}\" ")